packages/dara-components/dara/components/common/base_component.py

```diff
@@ -15,10 +15,50 @@
 limitations under the License.
 """
 
-from typing import Optional
+from typing import Optional, Literal
 
 from dara.core.definitions import ComponentInstance, StyledComponentInstance
 
+JustifyTypes = Literal[
```

Comment:
Same concepts as with enums, types should also be singular unless they represent a list of some sort, here's it a single value (but a union of possible options). So `JustifyType` and `AlignType`, or I could see an argument to just name it after the property name e.g. `JustifyContent` / `AlignItems`

Reply:
Oh ok, thank you!